### DISCUSSION TITLE
Inquiry Regarding Torch 2.1 Compatibility in MACE

### DISCUSSION THREAD

**Discussion by clecust** [ACEsuit/mace#270] [Q&A] [answered]
I recently came across the information in the README that MACE does not currently support Torch 2.1. I am interested in understanding the roadmap for MACE in terms of compatibility with Torch 2.1.

Could you please provide insights into when MACE is expected to release a version that supports Torch 2.1? 
Additionally, if I were to use the current version of MACE with Torch 2.1, are there expected problems such as bug in training models and interfacing with LAMMPS or OpenMM?

I appreciate your time and help in clarifying these questions. Thank you for your contributions.

**Comment by bernstei** [comment, marked as answer]
I'm pretty sure it actually works fine, and the docs just need to be updated
